linters/plugin.yaml

```diff
@@ -412,6 +412,12 @@ lint:
       extensions:
         - png
 
+    - name: postcss
+      extensions:
+        # TODO(Tyler): Do we want to include css here as well?
```

Comment:
don't think so? we have a `css` filetype already?

Reply:
Yeah, although there's an intersection. It may be more ergonomic to include it here with an inherits, but probably best to keep separate for extensibility